server.js

```diff
@@ -249,9 +249,18 @@ app
           .getJobs()
           .then((jobs) => {
             let filteredJobs = jobs.map((job) => {
+              let date = new Date(job.timestamp);
               return {
                 id: job.id,
                 title: _.get(job.data.details, bookTitle[req.query.queue_name]),
+                userName: job.data.userName,
```

Comment:
For older jobs, the username was not present. How are we handling that?

Reply:
By default, it shows empty text. But we can add a check and a placeholder (something like _No user info_) if required. What do you suggest?

Reply:
Let's show "-" for the older jobs.